roles/privacyidea/defaults/main.yml

```diff
@@ -1 +1,8 @@
 ---
+mariadb_root_password: ""
```

Comment:
I think the variables should have privacyidea in their name to prevent collisions with other variables